core/templates/services/i18n-language-code.service.ts

```diff
@@ -20,9 +20,18 @@ import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable, EventEmitter } from '@angular/core';
 import { AppConstants } from 'app.constants';
 
+/**
+ * Used to define if the translation key is type title or desciption.
+ */
+export enum TranslationKeyType {
+  Title = 'TITLE',
+  Description = 'DESCRIPTION',
```

Comment:
```suggestion
  TITLE = 'TITLE',
  DESCRIPTION = 'DESCRIPTION',
```
These members are constants.

Reply:
Done